metrotranscom/doorway
# feat: switch seed/migration scripts to use Bay Area jurisdiction

## Issue Overview

This PR addresses #issue


## Description

For #13,
* updating the migration directly to only insert the Bay Area Jurisdiction
  * meta thought: generally you dont update migrations directly but I think it's OK because we don't have any deployments of Doorway yet
* updating the Bloom seed scripts to only have one Jurisdiction, `Bay Area`
* updates the seeded listing/users/etc. to use the Bay Area jurisdiction. Some names and data are still very "alameda"-y, will tackle that in subsequent CL(s)

I still need to find more places with hardcoded Alameda stuff and/or fix tests if they are failing.


## How Can This Be Tested/Reviewed?

* `yarn db:reseed` with the new data
* update your public site .env file to be "Bay Area"
* confirm that localhost:3100/jurisdictions only has one jurisdiction
* confirm that you can see all of seeded listings in the public site.
![image](https://user-images.githubusercontent.com/10789523/226431147-98a61d73-77d3-424e-8dc4-e34ac935b9a9.png)


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
